cumulusci/tasks/bulkdata/delete.py

```diff
@@ -51,11 +56,24 @@ def _create_job(self, obj):
         self.bulk.close_job(delete_job)
         return delete_job
 
-    def _query_salesforce_for_records_to_delete(self, obj):
+    def compose_query(self, obj, criteria):
+        query = "select Id from {}".format(obj)
```

Comment:
Can we use typical SOQL query formatting, with keywords in uppercase (`SELECT Id FROM ...`)?

Reply:
be943daa57569c76ca1a48e8b234feade4de71a4